src/docs/metis.mdx

```diff
@@ -0,0 +1,163 @@
+    | <Copy value="0xf209815E595Cdf3ed0aAF9665b1772e608AB9380" label="0xf20...80" /> | <Reference label="StateCommitmentChain" url="https://github.com/MetisProtocol/mvm/blob/develop/packages/contracts/contracts/L1/rollup/StateCommitmentChain.sol" /> | The State Commitment Chain (SCC) contract contains a list of proposed state roots which Proposers assert to be a result of each transaction in the Canonical Transaction Chain (CTC). |
+    | <Copy value="0x081D1101855bD523bA69A9794e0217F0DB6323ff" label="0x081...ff" /> | <Reference label="L1CrossDomainMessenger" url="https://github.com/MetisProtocol/mvm/blob/develop/packages/contracts/contracts/L1/messaging/L1CrossDomainMessenger.sol" /> | The L1 Cross Domain Messenger contract sends messages from L1 to L2, and relays messages from L2 onto L1. |
+    | <Copy value="0x4200000000000000000000000000000000000007" label="0x420...07" /> | <Reference label="L2CrossDomainMessenger" url="https://github.com/MetisProtocol/mvm/blob/develop/packages/contracts/contracts/L2/messaging/L2CrossDomainMessenger.sol" /> | The L2 Cross Domain Messenger contract sends messages from L2 to L1, and relays messages from L1 onto L2. |
+    
+</Section>
+
+<Section title="RPC-API">
+
+    <Legend />
+
+    The following endpoints behave differently compared to the canonical Ethereum L1 implementation of the JSON RPC API.
+
+    | Method | Params | Rollup Behaviour | Ethereum L1 Behaviour |
+    | :----- | :----- | :--------------- | :-------------------- |
+    | `eth_getBlockByNumber` | Integer block number or string one of `safe`, `latest` or `pending` | Returns information for a given block.<br /><br />Adds additional information for transactions such as l1 block number, l1 timestamp, queue origin and raw transaction. | Returns information for a given block. <Modified /> |
+    | `eth_getBlockByHash` | hash | Returns information for a given block.<br /><br />Adds additional information for transactions such as l1 block number, l1 timestamp, queue origin and raw transaction. | Returns information for a given block. <Modified /> |
+    | `eth_getTransactionByHash` | hash | Returns information for a given transaction. <br /><br />Adds additional information such as l1 block number, l1 timestamp, queue origin and raw transaction. | Returns information for a given transaction. <Modified /> |
+    | `eth_getTransactionReceipt` | hash | Returns information for a given transaction. <br /><br />Adds additional information related to L1 calldata fees. | Returns information for a given transaction. <Modified /> |
+
+</Section>
+
+<Section title="Transaction Fees">
+
+    The L2 fees of Metis work identically to gas fees on Ethereum. Metis transactions must also, however, pay an L1 fee component to cover the cost of their calldata paid in METIS. You should use `eth_gasPrice` to determine the appropriate gas price needed.
```

Comment:
So the native currency of the rollup is METIS and fees are paid in METIS? If that's the case, let's reformat the message so that we say explicitly from the beginning that tx fees are paid in METIS and then clarify the case of the calldata costs. Whether they are part of the gas cost which is gas used * gas limit, meaning you simply have to set gas price high enough OR you set the gas price as `eth_gasPrice` tells you but you have to set some other property for the calldata fees

Reply:
The documentation says you only need to you the gas price as `eth_gasPrice` tells you: https://docs.metis.io/dev/protocol-in-detail/transaction-fees